qualang_tools/config/waveform_tools.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+# Definition of pulses follow Chen et al. PRL, 116, 020501 (2016)
+
+
+def drag_gaussian_pulse_waveforms(
+    amplitude, length, sigma, alpha, detune, delta, substracted
+):
+    """
+    Function that returns DRAG pulses that compensate for leakage and also for the AC stark shift
+    that the leakage creates to the qubit frequency.
+
+    Envelope: gaussian
+
+    :param amplitude: amplitude of voltage
+    :param length: pulse length
+    :param sigma: gaussian standard deviation
+    :param alpha: DRAG coefficient
+    :param detune: frequency shift to correct for AC start shift
+    :param delta: f_21 - f_10 - detune
+    :param substracted: if you would like to use a substracted gaussian
```

Comment:
:param bool substracted: If true, returns a substracted Gaussian to reduce high-frequency components due to the initial and final points offset. Default is true